cmd/argo/commands/cron/update.go

```diff
@@ -0,0 +1,105 @@
+package cron
+
+import (
+	"context"
+	"fmt"
+
+	"log"
+	"os"
+
+	"github.com/argoproj/pkg/errors"
+
+	"github.com/spf13/cobra"
+
+	"github.com/argoproj/argo-workflows/v3/cmd/argo/commands/client"
+	cronworkflowpkg "github.com/argoproj/argo-workflows/v3/pkg/apiclient/cronworkflow"
+	wfv1 "github.com/argoproj/argo-workflows/v3/pkg/apis/workflow/v1alpha1"
+	"github.com/argoproj/argo-workflows/v3/workflow/util"
+)
+
+type cliUpdateOpts struct {
+	output string // --output
+	strict bool   // --strict
+}
+
+func NewUpdateCommand() *cobra.Command {
+	var (
+		cliUpdateOpts  cliUpdateOpts
+		submitOpts     wfv1.SubmitOpts
+		parametersFile string
+	)
+	command := &cobra.Command{
+		Use:   "update FILE1 FILE2...",
+		Short: "update a cron workflow",
+		Run: func(cmd *cobra.Command, args []string) {
+			if len(args) == 0 {
+				cmd.HelpFunc()(cmd, args)
+				os.Exit(1)
+			}
+
+			if parametersFile != "" {
+				err := util.ReadParametersFile(parametersFile, &submitOpts)
+				errors.CheckError(err)
+			}
+
+			updateCronWorkflows(cmd.Context(), args, &cliUpdateOpts, &submitOpts)
```

Comment:
this perhaps should be split into a helper  as well as it's identical to `create` too